src/App.test.js

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
 import { shallow } from 'enzyme';
+import { Route } from 'react-router-dom';
 import App from './App';
+import Header from './components/Header';
+import Home from './components/Home';
```

Comment:
So, here is something that we've recently started to implement that will be beneficial for not just not but future as well. We can mock these imports by using ```jest.mock('./components/Header', () => 'Header');``` in lieu of your imports and it will achieve the same results 👍 . Reach out to `Gordon` for further clarity...

Reply:
@gordonhgraham

Reply:
good call out @kyleruschel, check out the `jest` documentation on mocks for the syntax.

The reason we do that is these unit tests don't care about the specific `header` component. they just care that it exists.